x-pack/plugins/apm/server/lib/anomaly_detection/create_anomaly_detection_jobs.ts

```diff
@@ -29,16 +29,12 @@ export async function createAnomalyDetectionJobs(
   const { ml, indices } = setup;
 
   if (!ml) {
-    throw new AnomalyDetectionError(ErrorCode.ML_NOT_AVAILABLE);
+    throw Boom.serverUnavailable(ML_ERRORS.ML_NOT_AVAILABLE);
```

Comment:
Use Boom errors over the custom `AnomalyDetectionError` (which has been removed everywhere)